src/fields/field.hh

```diff
@@ -55,8 +55,8 @@ public:
     typedef FieldAlgorithmBase<spacedim, Value> FieldBaseType;
     typedef std::shared_ptr< FieldBaseType > FieldBasePtr;
     typedef typename FieldAlgorithmBase<spacedim, Value>::Point Point;
+    typedef Value ValueType;
```

Comment:
This typedef is used, but same typedef in FieldAlgoBase is unnecessary - removed.